addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5174,6 +5174,14 @@ registry.layout_column = SnippetOptionWidget.extend({
         // Add the new column and update the grid height.
         rowEl.appendChild(newColumnEl);
         gridUtils._resizeGrid(rowEl);
+
+        // Scroll to the new column if it is not visible.
+        const newColumnPosition = newColumnEl.getBoundingClientRect();
+        const viewportHeight = this.$target[0].ownerDocument.body.clientHeight;
+        const topOffset = this._getTopOffset();
+        if (newColumnPosition.top < topOffset || newColumnPosition.bottom > viewportHeight) {
+            newColumnEl.scrollIntoView({ behavior: "smooth", block: "center" });
+        }
```

Comment:
Hmm indeed, I could just consider the middle in general instead of the top/bottom, but your idea seems better :slightly_smiling_face: 
For the record, I was tricked by the video in the task description, because it in fact adds three buttons but I only saw two at first, with the second one being the one with only the bottom half shown, so I thought it should scroll even if it was partially hidden :grimacing: 
But indeed, seeing the half should be enough :+1: